silverback/_cli.py

```diff
@@ -236,6 +241,42 @@ def workspaces(platform: PlatformClient):
         )
 
 
+@workspaces.command(name="new", section="Platform Commands (https://silverback.apeworx.io)")
+@click.option(
+    "-n",
+    "--name",
+    "workspace_name",
+    help="Name for new workspace",
+)
+@click.option(
+    "-s",
+    "--slug",
+    "workspace_slug",
+    help="Slug for new workspace",
+)
+@platform_client
+def new_workspace(
+    platform: PlatformClient,
+    workspace_name: str,
+    workspace_slug: str,
+):
+    """Create a new workspace"""
+
+    if workspace_name:
+        click.echo(f"name: {workspace_name}")
+        click.echo(f"slug: {workspace_slug or workspace_name.lower().replace(' ', '-')}")
+
+    elif workspace_slug:
+        click.echo(f"slug: {workspace_slug}")
+
```

Comment:
Probably can do this with clusters as well
```suggestion

    else:
        raise click.UsageError("Must provide a name or a slug/name combo")

```